File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;

class Main {
    
    static Scanner sc = new Scanner(System.in);
    
    public static void main(String[] args){
        int N = sc.nextInt();
        
        int Min = Integer.MAX_VALUE;
        int Max = Integer.MIN_VALUE;
        for(int n=0; n<N; n++){
            int num = sc.nextInt();
            Min = Math.min(Min, num);
            Max = Math.max(Max, num);
        }
        
        System.out.println(Min + " " + Max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
let N;
let Nums;
let Min = Number.MAX_SAFE_INTEGER;
let Max = Number.MIN_SAFE_INTEGER;

const main = () => {
    const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
    N = input[0];
    input.shift();
    Nums = input[0].split(' ');
    
    Nums.map(num => {
        Min = Math.min(Min, num);
    })
    Nums.map(num => {
        Max = Math.max(Max, num);
    })
    
    console.log(Min, Max);
}

main();